### DISCUSSION TITLE
Opposite of S-C

### DISCUSSION THREAD

**Discussion by phlawlessDevelopment** [helix-editor/helix#8391] [Ideas] [answered]
I'm a pretty clumsy typist (may be that's fairly rare in this community). When adding multiple cursors with S-C (C) I often overshoot the number of cursors I need (eg press the key 4 times when i only need 3 cursors. I can't find a way to remove the previously added cursor... I have the same issue with (x) and someone on Reddit helped me out with this config snippet '"X" = ["extend_line_up", "extend_to_line_bounds"]', but I can't seem to figure out how to do a similar thing with (C). I've been searching through the commands finder (<space>?) and the source code, but I can't even figure out where the multiple cursor command lives... Assuming that the cursors are just being stored in some sort of vector or similar DS, chances are I'd be able to add this myself if i could find the part of the code responsible for it. Any help would be greatly appreciated.

**Comment by sibouras** [comment, marked as answer]
`Alt-,`

**Comment by phlawlessDevelopment** [comment]
Aaaaaah YES ! Thank you so much ! That's exactly what I was looking for !

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by phlawlessDevelopment** [comment]
As an aside, I'm able to find A-, (remove_primary_selection) in the command finder, but still can't find S-C, do you happen to know what that might be called ? (I'd like to rebind this pair of commands to the up and down arrows)

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by sibouras** [reply]
its called copy_selection_on_next_line, you can find all of them in the docs https://docs.helix-editor.com/master/keymap.html#selection-manipulation

**Comment by phlawlessDevelopment** [comment]
Oh, I'm clearly an idiot.. I did see that, and (forgetting about the term yank) assumed that couldn't be the right thing... Thanks again.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
